sanssouci/reference_families.py

```diff
@@ -87,3 +87,93 @@ def inverse_beta_template(y, k, m):
 
     """
     return beta.cdf(y, k, m + 1 - k)
+
+
+def regularised_inverse_beta_template(y, k, m, reg=100):
+    """
+    Parameters
+    ----------
+    y : array of floats of shape (B, )
+        values to apply template to
+    k : int
+        index of rejection set in beta template
+    m : int
```

Comment:
is m differnt from len(y) ?

Reply:
Usually not, but it is less than m when working with the step down version of the algorithm (which is not implemented yet in python). So it's better to keep m as a parameter here.